### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by KseniyaKuzmina**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -3,23 +3,37 @@
...
-<!-- default file list -->
-*Files to look at*:
 
-* [Form1.cs](./CS/LineOfBestFit/Form1.cs) (VB: [Form1.vb](./VB/LineOfBestFit/Form1.vb))
-* [Program.cs](./CS/LineOfBestFit/Program.cs) (VB: [Program.vb](./VB/LineOfBestFit/Program.vb))
-<!-- default file list end -->
-# Obsolete - How to implement the best fit function, and apply it to a chart's series
+# How to Implement the Best Fit Function, and Apply it to a Chart's Series
+
+This example shows how to perform a custom regression analysis against scatter series points generated from a data table and plot the line of best fit.
```
Comment (arminal) [inline Readme.md]: plot the line of best fit? phrasing.

Comment (arminal) [reply Readme.md]: Also, can you tie the image below with this first sentence? Perhaps you could mention, "...as illustarted on the image below."

File: Readme.md [hunk 2]
```diff
@@ -3,23 +3,37 @@
...
-* [Program.cs](./CS/LineOfBestFit/Program.cs) (VB: [Program.vb](./VB/LineOfBestFit/Program.vb))
-<!-- default file list end -->
-# Obsolete - How to implement the best fit function, and apply it to a chart's series
+# How to Implement the Best Fit Function, and Apply it to a Chart's Series
+
+This example shows how to perform a custom regression analysis against scatter series points generated from a data table and plot the line of best fit. 
+
+![](images/line-of-best-fit.png)
+
+To do so, follow the steps below:
```
Comment (arminal) [inline Readme.md]: I don't think we need this sentence.

**Comment by arminal** [comment]
#GrammarReview

Even though this is a short doc, it needs another pass. The ideas don't flow well. See my edits.

File: Readme.md (hunk 1 above)
Comment (KseniyaKuzmina) [reply Readme.md]:
```suggestion
This example shows how to perform a custom regression analysis against scatter series points generated from a data table and display the Line of Best Fit as illustrated on the image below.
```
*Line of Best Fit* is a term here. https://www.investopedia.com/terms/l/line-of-best-fit.asp

File: Readme.md (hunk 2 above)
Comment (KseniyaKuzmina) [reply Readme.md]: Removed it.

File: Readme.md (hunk 1 above)
Comment (arminal) [reply Readme.md]: This example shows how to perform a custom regression analysis against scatter series points generated from a data table and how to display the Line of Best Fit as illustrated on the image below.

**Pull request merged**